src/inmanta/data/__init__.py

```diff
@@ -2554,13 +2478,17 @@ def set_connection_pool(pool):
         await cls.close_connection_pool()
 
 
+PACKAGE_WITH_UPDATE_FILES = inmanta.db.versions
+CORE_DATABASE_NAME = CORE_NAME
```

Comment:
CORE_DATABASE_NAME is a confusing name, since it's not the name of the database. Maybe better: CORE_SCHEMA_NAME,